full-service/src/service/account.rs

```diff
@@ -43,6 +46,12 @@ pub enum AccountServiceError {
 
     /// Invalid BIP39 english mnemonic: {0}
     InvalidMnemonic(String),
+
+    /// Error decoding base64 {0}
```

Comment:
Nit: Colon before error message
```suggestion
    /// Error decoding base64: {0}
```